src/containers/Downloads/DataSet/index.js

```diff
@@ -198,6 +196,34 @@ let DataSetErrorDownloading = ({
                 Try Again
               </Button>
             )}
+
+            <p>
+              If the problem persists, please contact{' '}
+              <a href="mailto:ccdl@alexslemonade.org" className="link">
+                ccdl@alexslemonade.org
+              </a>
+              {dataSet.failure_reason && (
+                <span>
+                  {' '}
+                  or{' '}
+                  <a
+                    href="https://github.com/AlexsLemonade/refinebio/issues/new"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    className="link"
+                  >
+                    file a ticket on Github
+                  </a>{' '}
+                  with the below error message:
```

Comment:
`following` is probably better than `below`. (I am not a native speaker of English either, you can double check with an expert like @jaclyn-taroni.)